core/src/test/java/org/pentaho/platform/util/StringUtilTest.java

```diff
@@ -75,14 +92,20 @@ public void testDoesPathContainParentPathSegment() {
     }
   }
 
-  public static void main( final String[] args ) {
-    StringUtilTest test = new StringUtilTest();
+  @Test
+  public void testMain() {
     try {
-      test.testTokenStringToArray();
-      test.testDoesPathContainParentPathSegment();
-
-    } finally {
+      Assert.assertTrue( true );
```

Comment:
This was a mistake on my part - I means to smoke test the StringUtil.main method to ensure that it does not throw any exceptions, this line should be added above the assert:
StringUtil.main( null );

The only reason for this test is to achieve 100% code coverage. I will be updating the PR with this change shortly.